src/components/common/CommentIconComponent.js

```diff
@@ -6,7 +6,7 @@ const CommentIconComponent = ({ commentCount }) => (
 <Fragment>
 <span className = 'icon-holder'>
 <img src = {CommentIcon} alt = 'CommentIcon' className= 'icon-color icon-height icon_margin'/>
-<sub className='count'>{commentCount}</sub>
+<sub className='count'>{commentCount === 0 ? '' : commentCount}</sub>
```

Comment:
Since `0` is falsy in javascript, it's better to write this as:
```suggestion
<sub className='count'>{commentCount || ''}</sub>
```